phalcon/Config.zep

```diff
@@ -215,8 +215,7 @@ class Config implements \ArrayAccess, \Countable
     {
         let index = strval(index);
 
-        //unset(this->{index});
-        let this->{index} = null;
+        unset this->{index};
```

Comment:
@niden @SidRoberts 
Compile error.
 [ERROR] Cannot use expression type: property-dynamic-access in "unset" in cphalcon/phalcon/Config.zep on line 218